src/entities/transactions/transaction.ts

```diff
@@ -8,42 +8,42 @@ import { Network } from '@rainbow-me/helpers/networkTypes';
 import { AddCashCurrencyAsset } from '@rainbow-me/references';
 
 export interface RainbowTransaction {
-  address: string;
-  balance: {
+  address?: string;
```

Comment:
@estebanmino @jxom Ive had to add optional parameters here as it looks like we use this function without alot of the params.

Reply:
oh yes, makes sense to me